pages.de/common/alias.md

```diff
@@ -0,0 +1,28 @@
+# alias
+
+> Erstellt Aliase - Wörter, die durch eine Befehlszeile ersetzt werden.
+> Aliase laufen mit der aktuellen Shell-Sitzung ab, es sei denn, sie sind in der Konfigurationsdatei der Shell definiert, z.B. `~/.bashrc`.
+
+- Listet alle Aliase auf:
+
+`alias`
+
+- Erstellt einen Alias:
+
+`alias {{wort}}="{{befehl}}"`
+
+- Zeigt den mit einem bestimmten Alias verknüpften Befehl an:
+
+`alias {{word}}`
+
+- Entferne einen Alias :
```

Comment:
```suggestion
- Entferne einen Alias:
```

Reply:
Those were my first command descriptions.
I try to be more thorough.
I don't want the description to include a person who executes the command, but only a description of what the command does.
The first ones are flawed. I'm correct them.

Reply:
@FrankBG67 I think it's fine either way. I just wanted it to be consistent, so that one form is used in all examples.